fay.cabal

```diff
@@ -158,7 +158,7 @@ library
     , time >= 1.4 && < 1.9
     , transformers >= 0.3 && < 0.4 || > 0.4.1 && < 0.7
     , transformers-compat >= 0.3 && < 0.6
-    , traverse-with-class >= 1.0 && < 2.0
+    , traverse-with-class >= 1.0 && < 1.1
```

Comment:
Updated this as well since e.g. a traverse-with-class v1.1 release could contain changes that break fay.